management.yml

```diff
@@ -3495,15 +3495,6 @@ paths:
       - MFA
       summary: Disable MFA for the current identity
       operationId: deleteMfa
-      parameters:
-      - description: An MFA validation request
-        name: mfaValidation
-        in: body
-        schema:
-          $ref: '#/definitions/mfaCode'
-      - type: string
```

Comment:
The header value is required to remove MFA from the current identity. The body was optional, and technically against the HTTP spec, and can be removed. However the header must stay.